### PR TITLE
dbt integration front page edits

### PR DESCRIPTION
**Pull request by sryza**
## Summary & Motivation

This is part of a larger set of software + docs changes for making Dagster more accessible to dbt users.

This makes some edits to the front page for the dbt integration, with a few higher-level aims:
- Get the value prop across ASAP
- Limit Dagster-specific terminology early on that might feel overwhelming
- Support different learning styles

Specific changes:
- Near the top, added an image & code sample of a dbt graph loaded into Dagster.
- Made some tweaks to the intro text based on my understanding of what language will get across best for learning Dagster + dbt users.
- Present the tutorial as one of a few different options for getting started with Dagster & dbt. When the dbt-focused Cloud NUX is ready, we can add that as another option.
- Added a link to a Dagster+dbt example project.
- Moved the section on how Dagster assets relate to dbt models farther down.

### CODE REVIEW

File: docs/content/integrations/dbt.mdx [hunk 1]
```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
   <a href="/integrations/dbt-cloud">dbt Cloud with Dagster guide</a>!
 </Note>
 
-Dagster orchestrates dbt alongside other technologies, so you can combine dbt with Spark, Python, etc. in a single workflow. Dagster's [software-defined asset](/concepts/assets/software-defined-assets) abstractions make it simple to define data assets that depend on specific dbt models, or to define the computation required to compute the sources that your dbt models depend on. You could, for example:
+Dagster orchestrates dbt alongside other technologies, so you can schedule dbt with Spark, Python, etc. in a single data pipeline.
 
-- Run your dbt models after ingesting data into your data warehouse
-- Selectively materialize dbt models and their dependencies
+Dagster's [software-defined asset](/concepts/assets/software-defined-assets) approach allows it to understand dbt at the level of individual dbt models. This means that you can:
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:15]: `it` feels awkward here - what is `it`? Dagster?

File: docs/content/integrations/dbt.mdx [hunk 2]
```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
...
 
-<DbtModelAssetExplanation />
+dbt_assets = dagster_dbt.load_assets_from_dbt_manifest("manifest.json")
 
-To learn how to load dbt models into Dagster as assets, check out the tutorial below or the quick version in the [dagster-dbt reference](/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project).
+
+@asset(compute_kind="tensorflow", non_argument_deps={"daily_order_summary"})
+def predicted_orders():
+    ...
+```
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:42]: Having the image and code block up against each other in the rendered page looks odd. Can we add some copy between these things to make it look less cluttered?

Ex:

```
Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that it's trained from:

[code example]

Which could produce an asset graph like the following:

[image]
```

Screen Shot 2023-06-26 at 11 15 21 AM

File: docs/content/integrations/dbt.mdx [hunk 3]
```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
...
-  <ArticleListItem
-    title="dagster-dbt integration reference"
-    href="/integrations/dbt/reference"
-  ></ArticleListItem>
-  <ArticleListItem
-    title="dagster-dbt API reference"
-    href="/\_apidocs/libraries/dagster-dbt"
-  ></ArticleListItem>
-</ArticleList>
+## Understanding the relationship between dbt models and Dagster Software-defined Assets
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:57]: This title is on the long side - can we shorten it up? Suggestions:

- Relating dbt models to Dagster Software-defined Assets
- Understanding how dbt models relate to Dagster Software-defined assets
- dbt models and Dagster Software-defined Assets (not my favorite, but it's the shortest)

File: docs/content/integrations/dbt.mdx (hunk 1 above)
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:15]:
```suggestion
Dagster's [Software-defined Asset](/concepts/assets/software-defined-assets) approach allows it to understand dbt at the level of individual dbt models. This means that you can:
```

File: docs/content/integrations/dbt.mdx [hunk 4]
```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
...
 
-Dagster orchestrates dbt alongside other technologies, so you can combine dbt with Spark, Python, etc. in a single workflow. Dagster's [software-defined asset](/concepts/assets/software-defined-assets) abstractions make it simple to define data assets that depend on specific dbt models, or to define the computation required to compute the sources that your dbt models depend on. You could, for example:
+Dagster orchestrates dbt alongside other technologies, so you can schedule dbt with Spark, Python, etc. in a single data pipeline.
 
-- Run your dbt models after ingesting data into your data warehouse
-- Selectively materialize dbt models and their dependencies
+Dagster's [software-defined asset](/concepts/assets/software-defined-assets) approach allows it to understand dbt at the level of individual dbt models. This means that you can:
 
-Dagster has built-in support for loading dbt models, seeds, and snapshots as software-defined assets, enabling you to:
+- Use Dagster's UI or APIs to run subsets of your dbt models, seeds and snapshots.
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:17]:
```suggestion
- Use Dagster's UI or APIs to run subsets of your dbt models, seeds, and snapshots.
```

Please use Oxford/serial commas

File: docs/content/integrations/dbt.mdx [hunk 5]
```diff
@@ -10,69 +10,52 @@ description: Dagster can orchestrate dbt alongside other technologies.
...
+- Track failures, logs, and run history for individual dbt models, seeds, and snapshots.
+- Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that its trained from.
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:19]:
```suggestion
- Define dependencies between individual dbt models and other data assets. For example, put dbt models after the Fivetran-ingested table that they read from, or put a machine learning after the dbt models that it's trained from.
```

This should be a contraction, not a possessive

**Review by erinkcochran87 — COMMENTED** [review]
Back to you!

File: docs/content/integrations/dbt.mdx (hunk 2 above)
Comment (sryza) [reply docs/content/integrations/dbt.mdx:42]: Yeah it does look odd. I was hoping to get the image to show up above the fold. I'm pushing up a version that includes text in between but puts the image first.  Let me know what you think.

File: docs/content/integrations/dbt.mdx (hunk 3 above)
Comment (sryza) [reply docs/content/integrations/dbt.mdx:57]: I like "Understanding how dbt models relate to Dagster Software-defined assets"

File: docs/content/integrations/dbt.mdx [hunk 6]
```diff
@@ -10,69 +10,56 @@ description: Dagster can orchestrate dbt alongside other technologies.
...
 
-## Using dbt with Dagster
+<Image
+alt="Dagster graph with dbt, Fivetran, and TensorFlow"
+src="/images/integrations/dbt/dagster-dbt-fivetran-tensorflow.png"
+width={1834}
+height={1220}
+/>
+
+Can be produced from code like this:
```
Comment (erinkcochran87) [inline docs/content/integrations/dbt.mdx:32]: <3 ty!

File: docs/content/integrations/dbt.mdx (hunk 2 above)
Comment (erinkcochran87) [reply docs/content/integrations/dbt.mdx]: It looks great, thanks so much for making this change :)

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**